content/knowledge-base/integrations/vault-as-secret-store.md

```diff
@@ -56,22 +56,30 @@ for the cluster where Crossplane is running.
 
 1. Install Vault Helm Chart
 
+Add the Vault Helm chart.
 ```shell
-kubectl create ns vault-system
+helm repo add hashicorp https://helm.releases.hashicorp.com --force-update --create-namespace
```

Comment:
I think we want `—create-namespace` on the install command rather than repo add

Reply:
and this is why we have code reviews.